clients/java/signalr/src/main/java/InvocationMessage.java

```diff
@@ -1,8 +1,8 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-public class InvocationMessage {
-    int type = 1;
+public class InvocationMessage extends HubMessage {
+    int type = HubMessageType.INVOCATION.value;
```

Comment:
I don't see where type is used.

Reply:
We need this field for when we serialize the invocation message and send it.